packages/katello/katello/hostname-change.rb

```diff
@@ -167,6 +167,12 @@ def using_custom_certs?(scenario_answers)
       scenario_answers["certs"]["server_cert_req"]
     end
 
+    def delete_puppet_certs
+      puppet_ssldir = @scenario_answers['puppet']['ssldir']
+
+      run_cmd("rm -rf '#{puppet_ssldir}'")
```

Comment:
Updated this to remove the whole folder. It appears to work! I tested it on a nightly proxy. @johnpmitsch can you confirm it works on your katello nightly?